README.md

```diff
@@ -101,6 +101,19 @@ meson _build --prefix=/usr/local
 ninja -C _build install
 ```
 
+## :electric_plug: Podman Socket Activation
+
+To connect to the local Podman instance, the systemd socket must be enabled. You can enable it permanently by issuing the following commands:
+
+```shell
+$ systemctl --user enable --now podman.socket
+$ ls $XDG_RUNTIME_DIR/podman/podman.sock
+/run/user/1000/podman/podman.sock
+$ export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
```

Comment:
Not sure about this line. Isn't this just relevant for the current shell? In this specific case, you'd set this environment variable to use docker specific commands right after. 

I think it can be removed as it shouldn't have any effect in Pods.

Btw, You don't need to open new PRs for change requests.
In your case, the PR has just one commit. So you can just apply the changes from the review locally and add them to the staging area
```shell
$ git add README.md
```
then use
```shell
$ git commit --amend
```
to change your PR commit locally. Here you can also change the commit message as requested in the other PR.

Finally, you can force push the commit by issuing
```shell
$ git push --force
```

This will then overwrite the commit in the PR with your local commit.

Reply:
oh I don't use git to do my pull requests. I just do pull requests on the github website.

Reply:
Never mind. You can open a new PR in this case. Please use `chore(readme): Add socket activation instructions
` as commit message.